eventhub/Module.md

```diff
@@ -246,100 +224,46 @@ Note:
 You can specify the event hub path as a parameter of the deleteEventHub method.
 This operation will return a ballerina error if the operation failed.
 
-# Samples:
+## Snippets
+Snippets of some operations.
 
 1. Sending an event.
 
 ```ballerina
-import ballerinax/azure_eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
    var result = eventHubClient->send("myhub", "eventData");
-}
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_event.bal
 
 2. Sending an event with broker properties and user properties.
 
 ```ballerina
-import ballerinax/azure_eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
    map<string> brokerProps = {"CorrelationId": "32119834", "CorrelationId2": "32119834"};
    map<string> userProps = {Alert: "windy", warning: "true"};
 
    var result = eventHubClient->send("myhub", "eventData", userProps, brokerProps);
-}
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_event_with_broker_and_user_properties.bal
 
 3. Sending an event with broker properties, user properties & partition key.
```

Comment:
Same here. Please fix in other similar occurences too.

Reply:
Fixed in df7c75b . Thank you.